app/assets/stylesheets/pages/_submission-form-page.scss

```diff
@@ -14,20 +14,91 @@
   }
 }
 
-.header-image {
-  text-align: center;
-  img{
-    display:none;
-    @include media($tablet) {
-      display: inline;
-      margin-bottom: -5%;
-    }
+.hero-section {
+  padding: 2rem 0;
+
+  @include media($tablet) {
+    padding: 5rem 0;
+    background: image-url('cover.svg') no-repeat center top;
+    background-size: auto 100%;
+    margin-bottom: -2rem;
+    position: relative;
+    z-index: 1;
   }
 }
 
-.pattern-wrapper{
-  @include media(1350px) {
-    background: image-url('pattern.png');
-    background-size: 100%;
+.hero-section-content {
+  display: flex;
```

Comment:
We don't have any mixin nor autoprefixer in the repo, so it can be broken in some browsers http://shouldiprefix.com/#flexbox